presentation/Main.java

```diff
@@ -52,10 +67,37 @@ public static void main(String[] args) {
                     );
                     System.out.println();
                 }
+                case Menu.SHOW_AND_SORT_STUDENTS -> {
+                    System.out.println("1: Sort by Student Id");
+                    System.out.println("2: Sort by First Name");
+                    System.out.println("3: Sort by Last Name");
+                    System.out.print("Enter your choice: ");
+                    int choice = scanner.nextInt();
+                    List<Student> students = studentService.getAllStudents();
+                    System.out.printf("%-15s%-15s%-15s%n","Student Id","First Name","Last Name");
+                    switch (choice){
+                        case 1 -> students.stream().sorted(Comparator.comparing(Student::getStudentId)).forEach(student ->
```

Comment:
Here you could also create one object (e.g. `List<Student> sortedStudents`), and save the sorted students inside this list based on the correct property the user wants to sort (e.g. `sortedStudents =students.stream().sorted(Comparator.comparing(Student::getFirstName);`). After you applied the correct property to sort (id, first name, or last name), you can use `.forEach(student -> ...)` to print the result(s).